plugins/main/server/types.ts

```diff
@@ -17,11 +17,13 @@
  * under the License.
  */
 
+import { WazuhCore } from '../common/types';
+
 export interface WazuhPluginSetup {}
 
 export interface WazuhPluginStart {}
 
 export type PluginSetup = {
   securityDashboards?: {}; // TODO: Add OpenSearch Dashboards Security interface
-  wazuhCore: {};
+  wazuhCore: WazuhCore;
```

Comment:
**nitpick**: this should use the type of wazuhCore plugin instead of defining a type in the `wazuh` plugin